cmd/check-gh-automation/main.go

```diff
@@ -196,7 +196,11 @@ func gatherModifiedRepos(releaseRepoPath string, logger *logrus.Entry) []string
 	for _, c := range configs {
 		path := strings.TrimPrefix(c, config.CiopConfigInRepoPath+"/")
 		split := strings.Split(path, "/")
-		orgRepos.Insert(fmt.Sprintf("%s/%s", split[0], split[1]))
+		// Check if this is a config file like ".config.prowgen" rather than a repo
+		isConfigFile := strings.HasPrefix(split[1], ".")
```

Comment:
We could just `continue` if the name is `.config.prowgen`

Reply:
Yeah, thats fine too. I wanted to avoid issues if we created other types of config files in the future, but I suppose that is not very likely so I can just tailor it to this case.